### PR TITLE
fix history and themes

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

File: card/mod/03_machines/set/type/scss.rb [hunk 1]
```diff
@@ -4,12 +4,11 @@ def diff_args
   {:format=>:text}
 end  
 
-
 format do
   include Css::Format
   
   view :core do |args|
-    process_content compile_scss(_render_raw)
+    compile_scss(process_content _render_raw)
```
Comment (xithan) [inline card/mod/03_machines/set/type/scss.rb]: make inclusions work in css and scss cards

File: card/mod/01_history/set/all/history.rb [hunk 2]
```diff
@@ -40,14 +40,13 @@ def create_act_and_action
...
     Action.fetch(a_id) || nil
   end
-  rollback_actions.each do |action|  
+  rollback_actions.each do |action|
     if action.card_id == id
-      revision.merge!(revision(action)) 
+      revision.merge!(revision(action))
     else
-      revision[:subcards].merge!(revision(action))
+      revision[:subcards][action.card.name] = revision(action)
```
Comment (xithan) [inline card/mod/01_history/set/all/history.rb]: this was the reason that for every db field a card was created

**Comment by xithan** [comment]
maybe useful to review this PR: add ?w=1 to the url to hide whitespace changes

File: card/mod/03_machines/set/type/scss.rb (hunk 1 above)
Comment (ethn) [inline card/mod/03_machines/set/type/scss.rb]: I feel like there was a rationale for the previous order, but I can't remember what now.  Perhaps it was something like wanting to be able to click "add +image" in the middle of an html rendering of a css file?

At any rate, I think for now this is better.

Comment (ethn) [inline card/mod/03_machines/set/type/scss.rb]: And if not, it's my fault for not writing a test way back when ;)

File: card/mod/06_bootstrap/set/self/bootswatch_mixins.rb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+view :raw do |args|
+  File.read "#{Cardio.gem_root}/mod/06_bootstrap/lib/stylesheets/#{card.codename}.css"
+end
+
+format :html do
+  view :editor do |args|
+    "Content is stored in file and can't be edited."
+  end
+end
```
Comment (ethn) [inline card/mod/06_bootstrap/set/self/bootswatch_mixins.rb]: I gather this isn't used yet here (but is in the other PR)?

File: card/mod/06_bootstrap/set/all/rich_bootstrap.rb [hunk 4]
```diff
@@ -1,7 +1,7 @@
 format :html do
   
-  def glyphicon icon_type
-    content_tag(:span, '', :class=>"glyphicon glyphicon-#{icon_type}", 'aria-hidden'=>true)
+  def glyphicon icon_type, extra_class=''
+    content_tag(:span, '', :class=>"glyphicon glyphicon-#{icon_type} #{extra_class}", 'aria-hidden'=>true)
   end
```
Comment (ethn) [inline card/mod/06_bootstrap/set/all/rich_bootstrap.rb]: we really need a more general solution for classes!

**Pull request merged**